csharp/src/AddressBook/Addressbook.cs

```diff
@@ -62,7 +62,7 @@ public static partial class Addressbook {
   }
   #region Messages
   [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
-  public sealed partial class Person : pb::IMessage<Person>, global::System.IEquatable<Person> {
+  public sealed partial class Person : pb::IMessage<Person>, global::System.IEquatable<Person>, pb::IDeepCloneable<Person> {
```

Comment:
Any reason why IMessage doesn't inherit from IDeepCloneable?  Looks like every message should be cloneable. (btw, same question for Equatable - but I think the decision to have separate IEquatable comes from times where you had an options to disable generating equals and hashCode in the codegen).